docs/modules/plugins/pages/plugin-web-app.adoc

```diff
@@ -253,6 +253,100 @@ include::partial$dynamic-variables-ui.adoc[]
 
 == Profiles
 
+Profiles allow to choose or parameterize browser that tests run on. See more details on the profiles and nesting strategy on the xref:ROOT:tests-configuration.adoc#_profiles[configuration page].
+
+Web-app plugin contains significant amount of built-in profiles. They can be found after plugin installation by the following path:
+[source, DIR]
+----
+Referenced Libraries -> vividus-plugin-web-app-version -> properties -> profile -> web
+----
+
+Let's go through the structure and review its content.
+
+=== Desktop
+
+General place for web profiles. Each of the directory contains browser related properties. This is a place for adjustments for your browser. Add there any properties you need: screen resolution, path to webdriver or to a custom browser.
+[cols="~a,~a", options="header"]
+|===
+|Profile
+|Example of properties
+
+.^|web/desktop/chrome
```

Comment:
as I see the properties were described without respect to another properties that we gather along the way from web and desktop properties, at the same time I'm not sure about whether we need all of them, but such things like platform name and platform version are 100% worth mentioning